html/janus.js

```diff
@@ -2384,26 +2387,35 @@ function Janus(gatewayCallbacks) {
 						pluginHandle.consentDialog(true);
 					}
 					let constraints = Janus.trackConstraints(track), stream = null;
-					if(track.type === 'audio' || track.type === 'video') {
+					if(track.type === 'audio' || track.type === 'video' || track.type === 'screen') {
+						// Set if User Media should be used of if it should be Display Media
+						let captureUserMedia = track.type !== 'screen';
```

Comment:
Please use brackets to avoid ambiguity and improve readability. As a side note, please write `getUserMedia` and `getDisplayMedia` in the comment above that, not "User Media" and "Display Media".